exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/grpc/DefaultGrpcExporter.java

```diff
@@ -33,6 +34,9 @@
 
   private final ThrottlingLogger logger = new ThrottlingLogger(internalLogger);
 
+  // We only log unavailable once since it's a configuration issue that won't be recovered.
```

Comment:
couldn't someone fix their collector and things would start working again?

Reply:
Yes, although probably rare. Spamming the logs is probably not useful in most situations.

Reply:
well..it won't be spammed, exactly, since we're throttling the logging. I just wonder about the one message getting lost and people not seeing it vs. having a few messages per minute that remind them things are broken with their collector and/or their configuration.

Reply:
IMO since it's SEVERE, I think a single log is better even if throttled - it's similar to any sort of logging we'd do in the SDK configuration codepath, and for SEVERE I guess we'd expect it to get noticed.

Reply:
I guess we'll wait and see how many confused users we get and we can adjust as needed.